test/integration/server/inspec/server_spec.rb

```diff
@@ -36,32 +33,46 @@
 
 describe file('/root/.my.cnf') do
   it { should be_file }
-  its(:content) { should match(/jzYY0cQUnPAMcqvIxYaC/) }
+  its('content') { should match(/jzYY0cQUnPAMcqvIxYaC/) }
 end
 
 describe command('sysctl vm.swappiness') do
```

Comment:
I think this can be replaced by the [kernel_parameter](https://www.inspec.io/docs/reference/resources/kernel_parameter/) resource.